.github/workflows/sub_buildUbuntu2204.yml

```diff
@@ -135,7 +135,8 @@ jobs:
                             shiboken2                                   \
                             swig                                        \
                             ccache                                      \
-                            xvfb
+                            xvfb                                        \
+                            libfmt-dev
```

Comment:
Are these dependencies that we need to let downstream packagers know about once this PR is merged?

Reply:
It has been the subject of quite some thought.

Finally, @chennes managed to have cmake download the library from github, build it, and then build FreeCAD:
https://github.com/FreeCAD/FreeCAD/pull/8508

libfmt, when available as a package, is generally a library to be statically linked (not shared), so it becomes part of the code of the binary.

I imagine (do not hold me for an expert) downstream packagers will just provide a source package with what we have, including the magic in cmake, and will build the binary from it. The only issue I can think about is, if the computer building the packages does not have access to internet for cmake (some kind of limitation/protection) and cannot download the dependency. We may ask Kurt, he will know at least the Debian side of the story.